mmv1/products/compute/NetworkEndpoints.yaml

```diff
@@ -0,0 +1,139 @@
+# Copyright 2023 Google Inc.
+# Licensed under the Apache License, Version 2.0 (the "License");
+# you may not use this file except in compliance with the License.
+# You may obtain a copy of the License at
+#
+#     http://www.apache.org/licenses/LICENSE-2.0
+#
+# Unless required by applicable law or agreed to in writing, software
+# distributed under the License is distributed on an "AS IS" BASIS,
+# WITHOUT WARRANTIES OR CONDITIONS OF ANY KIND, either express or implied.
+# See the License for the specific language governing permissions and
+# limitations under the License.
+
+--- !ruby/object:Api::Resource
+name: 'NetworkEndpoints'
+kind: 'compute#networkEndpoints'
+base_url: 'projects/{{project}}/zones/{{zone}}/networkEndpointGroups/{{network_endpoint_group}}'
+description: |
+  A set of network endpoints belonging to a network endpoint group (NEG). A
+  single network endpoint represents a IP address and port combination that is
+  part of a specific (NEG). NEGs are zonal collections of these endpoints for
+  GCP resources within a single subnet. **NOTE**: Network endpoints cannot be
+  created outside of a network endpoint group.
+
+  This resource is authoritative for a single NEG. Any endpoints not specified
+  by this resource will be deleted when the resource configuration is applied.
+immutable: true
+create_verb: :POST
+create_url: projects/{{project}}/zones/{{zone}}/networkEndpointGroups/{{network_endpoint_group}}/attachNetworkEndpoints
+delete_verb: :POST
+delete_url: projects/{{project}}/zones/{{zone}}/networkEndpointGroups/{{network_endpoint_group}}/detachNetworkEndpoints
+read_verb: :POST
+self_link: projects/{{project}}/zones/{{zone}}/networkEndpointGroups/{{network_endpoint_group}}/listNetworkEndpoints
+identity:
+  - networkEndpointGroup
+collection_url_key: 'items'
```

Comment:
```suggestion
```

This is only used during [sweeper](https://github.com/hashicorp/terraform-provider-google/wiki/Developer-Best-Practices#sweepers) generation, and this resource won't generate a sweeper because it has a custom pre_delete. (which is fine, I am guessing any dangling resources should get cleaned up by the sweeper for the NEG.)

Reply:
Good to know, removed.